com.discordapp.Discord.json

```
@@ -54,22 +54,51 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2",
-                    "sha256": "8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753"
+                    "url": "https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.5.tar.bz2",
+                    "sha256": "9b92ed0316bf2486121d8bac88bd1878f16b43bd335f18009b1f941f1eca93a1",
+                    "x-checker-data": {
+                        "type": "anitya",
+	                "project-id": "14958",
+	                "url-template": "https://xorg.freedesktop.org/archive/individual/app/xprop-$version.tar.bz2"
+                    }
                 }
             ]
         },
+	{
+	  "name": "python-setuptools_scm",
+	  "buildsystem": "simple",
+	  "build-commands": [
+	    "python3 setup.py install --prefix=/app --root=/"
```

Use pip and same args as in https://github.com/flatpak/flatpak-builder-tools/blob/master/pip/flatpak-pip-generator

```suggestion
	    "pip3 install --no-deps --no-build-isolation --verbose --prefix=${FLATPAK_DEST} ."
```